pkg/cmd/server/server.go

```diff
@@ -501,6 +514,16 @@ func (s *server) runControllers(config *api.Config) error {
 		s.logger,
 	)
 
+	go func() {
+		metricsMux := http.NewServeMux()
+		metricsMux.Handle("/metrics", promhttp.Handler())
+		if err := http.ListenAndServe(s.metricsAddress, metricsMux); err != nil {
```

Comment:
Small nit: It's nice to log the address/port that the metrics are using. It's helpful in troubleshooting them not working.